lib/controller.js

```diff
@@ -252,7 +252,7 @@ Tessel.get = function(options) {
                         // show "(not authorized)" if there is a lanConnection
                         /* istanbul ignore next */
                         const authorization = isAuthorized || !!tessel.usbConnection ? '' : '(not authorized)';
-                        const display = `\t${tessel.connection.connectionType}\t${colors.magenta(tessel.name)}\t${authorization}`;
+                        const display = `\t${tessel.connection.connectionType}\t${tessel.name}\t${authorization}`;
```

Comment:
Did you mean to change to `.displayName` here as well?

Reply:
Hm... yep! I'll fix that.